\chapter{The Structural Flexibility of \protein{Mad1} Facilitates the Assembly of the Mitotic Checkpoint Complex (MCC)}
\label{chpt:4}

In previous chapters, we presented evidence on the cooperativity among multiple MELT motifs on the same \protein{Knl1} phosphodomain (the first layer in the SAC signaling cascade). This phenomenon is probably explained by SAC proteins that concurrently bind to these MELT motifs in close spatial proximity (the second layer). In this chapter, we will focus on the spatio-temporal coupling scaffolded by \protein{Mad1} between \protein{Mad2}'s conformational change and the assembly of the MCC \cite{BUB1-CDC20-MAD1,Tripartite} (note that as mentioned in \myref{CoreSAC}, ``\protein{Mad1}'' here and in many other instances throughout this chapter refers to the \protein{Mad1}-\protein{Mad2} heterotetramer). We term it ``the third layer'' in the SAC signaling cascade. % As mentioned in \myref{Chapter3Discussions}, \protein{Mad1} can be recruited to signaling kinetochores by multiple pathway.

The formation of the \protein{Cdc20}-\protein{Mad2} dimer, an MCC subcomplex, is considered to be the rate-limiting step in the assembly of the MCC based on an \Latin{in vitro} study \cite{Faesen2017}. How this reaction is catalyzed in the cell was unclear for a long time. Critical studies published later showed that locking cytosolic \protein{Mad2} to the closed conformation inhibited the heterodimerization between \protein{Cdc20} and \protein{Mad2} and compromised the SAC signaling activity \cite{Ma+Poon2016,Ma+Poon2018,Kim2018}. These findings hinted that the conformational change of \protein{Mad2} and the formation of the \protein{Cdc20}-\protein{Mad2} heterodimer might be temporally coupled. In 2021, two papers published back-to-back (based on either \Latin{in vitro} reconstitution \cite{BUB1-CDC20-MAD1} or studies in \Latin{C. elegans} \cite{Tripartite}) presented evidence supporting the cooperative binding of \protein{Cdc20} to signaling kinetochores and the spatio-temporal coupling between \protein{Mad2}'s conformational change and the formation of the \protein{Cdc20}-\protein{Mad2} dimer (see \myref{TwoModels}), which offered critical molecular insight into the synergy in this process.

In this chapter, we approached the suggested spatio-temporal coupling from a different angle -- by looking into the role of the conserved structural flexibility of \protein{Mad1} in this process. We began this hypothesis-driven study independently in 2018, which has evolved into a collaboration project. I am the sole contributor to most cell biology data here. Simon Han performed preliminary tests on the feasibility and physiological activity of internally tagged Mad2p in the budding yeast. We took advantage of the free ColabFold advanced Jupyter Notebook (henceforth simply referred to as ``ColabFold advanced algorithm''), which implemented the tremendously successful, recently publicized structural prediction algorithm AlphaFold 2 in Google Colaboratory \cite{ColabFold, AlphaFold}. \Latin{In vitro} reconstitution results from our collaborators (Dr. Valentina Piano and Amal Alex from Dr. Andrea Musacchio's group at the Max Planck Institute of Molecular Physiology in Germany) will be mentioned to write a coherent story, but those data are not included here.

\section{The molecular mechanism of the formation of \protein{Cdc20}-\protein{Mad2}: the ``assembly line'' model \Latin{versus} the ``knitting'' model}
\label{TwoModels}

The formation of the \protein{Cdc20}-\protein{Mad2} dimer, an MCC subcomplex, is considered to be the rate-limiting step in the assembly of the MCC based on an \Latin{in vitro} study \cite{Faesen2017}. However, how the formation of \protein{Cdc20}-\protein{Mad2} is catalyzed in the cell was unclear for a long time. \protein{Cdc20} has a flexible N-terminal region with a \protein{Mad2}-interacting motif (MIM) and a C-terminal WD40 \textbeta{}-propeller fold \cite{CDC20-KEN} (in \myref{TwoModelCartoons}, the N- and C-terminal regions are represented by a light gray line and a light gray circle, respectively). \protein{Mad2} has three known conformations: the open conformation (denoted as O-\protein{Mad2} and illustrated as a magenta open lock with a circular body in \myref{TwoModelCartoons}), the intermediate conformation (denoted as I-\protein{Mad2} and illustrated as a peach open lock with a rectangular body in \myref{TwoModelCartoons}), and closed (denoted as C-\protein{Mad2} and illustrated as a red closed lock with a rectangular body in \myref{TwoModelCartoons}) \cite{I-MAD2}. A popular explanation of the catalytic mechanism of the switch of \protein{Mad2} from the open conformation to the closed conformation is the template model \cite{TemplateModel}. This model states that the conformational switch of \protein{Mad2} may be facilitated by the homodimerization between a closed \protein{Mad2} (bound to \protein{Mad1}'s MIM in the \protein{Mad1}-\protein{Mad2} heterotetramer) and the \protein{Mad2} that undergoes the conformational switch. C-\protein{Mad2} binds to \protein{Cdc20}, buckling the MIM of \protein{Cdc20} in a safety belt-like manner similar to how C-\protein{Mad2} binds to \protein{Mad1} \cite{Structure1GO4, SpMCC}. The \protein{Cdc20}-\protein{Mad2} dimer binds to \protein{BubR1}/\protein{BubR1}-\protein{Bub3} and forms the MCC.

\protein{Cdc20} binds to \protein{Mad1}'s C-terminal RING finger-containing proteins, WD repeat-containing proteins, and DEAD-like helicases (RWD) domain phosphorylated by \protein{Mps1} through BOX1, an N-terminal basic motif of \protein{Cdc20} \cite{Ji2017eLife, BUB1-CDC20-MAD1} (illustrated in \myref{TwoModelCartoons} as the contact between \protein{Cdc20}'s light gray N-terminal region and \protein{Mad1}'s dark green C-terminal domain). Given the spatial proximity between the catalytic site of \protein{Mad2}'s conformational switch and the MIM of \protein{Cdc20}, it is attractive to investigate the potential coupling between \protein{Mad2}'s conformational switch (\textcircled{\small{1}} in \myref{TwoModelCartoons}) and the binding between \protein{Cdc20} and \protein{Mad2} (\textcircled{\small{2}} in \myref{TwoModelCartoons}). Therefore, we set out to test two opposite models, which we named the ``assembly line'' model and the ``knitting'' model. As the names suggest, the ``assembly line'' model indicates that the two processes happen in two separate steps and newly switched \protein{Mad2} from step \textcircled{\small{1}} enters a free cytosolic pool and then participates in step \textcircled{\small{2}}, as if two steps in an assembly line mediated by a warehouse. In contrast, the ``knitting'' model indicates that like knitting, the two processes are closely coordinated in one step  (\Latin{i.e.}, \textcircled{\small{1}} is coupled to \textcircled{\small{2}}). The two regions of the \protein{Mad1}-\protein{Mad2} heterotetramer with known structures shown in the top panel of \myref{HsMAD1CTerStructure} function as the two knitting needles. They are connected by a region of unknown structure, which is predicted to be a flexible loop (see \myref{MAD1_MIM-RLK_Lupas+DeepCoil2}; this region is henceforth simply referred to as ``\protein{Mad1}'s loop region''). Interestingly, the primary sequence of this loop region is not well conserved from yeast to human, but the secondary structure is predicted to be conserved.

\begin{figure}
    \centering
    \includegraphics[width=0.75\textwidth]{chapters/figures/MAD1C+KnittingModel.pdf}
    \phantomsubfiglabel{HsMAD1CTerStructure} % subfigure A
    \phantomsubfiglabel{TwoModelCartoons} % subfigure B
    \caption{\textbf{The structure of the \protein{Mad1}-\protein{Mad2} heterotetramer and the two models of the molecular mechanism of the formation of \protein{Cdc20}-\protein{Mad2}.}}
    \noindent\justifying (A) Top panel: the known structure of the \protein{Mad1}-\protein{Mad2} heterotetramer. PDB IDs: 1GO4 (left, \cite{Structure1GO4}), 4DZO (right, \cite{Structure4DZO}). Structures of the N-terminus and the segment spanning a.a. 580--597 are unknown. Various secondary structure prediction algorithms consistently predicted this segment to be a flexible loop (though the predicted starting and ending positions may vary among different algorithms). Bottom panel: a cartoon illustrating the known structure of the \protein{Mad1}-\protein{Mad2} heterotetramer. The color scheme matches the top panel. To distinguish the two \protein{Mad1} proteins [and their C-terminal RWD domains] and the two \protein{Mad2} proteins, slightly different colors are applied. In all cartoons later, both copies will use the same color scheme. The sequence of the loop region (a.a. 582--600) predicted by Lupas' method is shown \cite{LupasCOILS}. Serine/threonine residues are colored green. Proline residues are colored orange. Negatively charged residues are colored red. Lysine residues are colored blue. All cartoons in this chapter are not to scale. (B) The two models of the molecular mechanism of the formation of \protein{Cdc20}-\protein{Mad2}: the "assembly line" model (left) and the "knitting" model (right).
    \label{MAD1C+KnittingModel}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.89\textwidth]{chapters/figures/MAD1_MIM-RLK_Lupas+DeepCoil2.pdf}
    \caption{\textbf{The secondary structure of \protein{Mad1}'s loop region is well conserved.}}
    \noindent\justifying Although the primary sequence of \protein{Mad1}'s loop region is not conserved, proline residues are commonly (not always) found in this region. The figure shows coiled-coil predictions by two algorithms (black curves: Lupas' method using a gliding window size of 28 residues \cite{LupasCOILS}; blue curves: raw predicted probabilities by DeepCoil2 \cite{DeepCoil}) on the region spanning from \protein{Mad1}'s MIM (which is also not a coiled coil \cite{Structure1GO4}) to \protein{Mad1}'s consensus RLK motif from \Latin{Homo sapiens} (human), \Latin{Danio rerio} (zebrafish), \Latin{Xenopus Laevis} (African clawed frog), \Latin{Saccharomyces cerevisiae} (budding yeast), and \Latin{Schizosaccharomyces pombe} (fission yeast). The $x$-axis represents the coordinates of residues within \protein{Mad1}. The primary sequences of full-length \protein{Mad1} proteins were supplied as the input, but only probability predictions on this region are shown here. The RLK motif directly binds to \protein{Bub1} \cite{Ji2017eLife, BUB1CD1-MAD1CStructure} and is located within the coiled coil leading to the RWD domain (see the crystal structure on the right in \myref{HsMAD1CTerStructure}). Residues within the putative loop region are colored as in \myref{HsMAD1CTerStructure}. Similar prediction results were obtained using MARCOIL \cite{MARCOIL} and PSIPRED 4.0 \cite{PSIPREDOriginal, PSIPREDPlatform} although the exact starting and ending residues of the flexible loop may differ (data not shown).
    \label{MAD1_MIM-RLK_Lupas+DeepCoil2}
\end{figure}

We reason that the ``knitting'' model is more realistic based on many pieces of evidence from previous literature. First, locking cytosolic \protein{Mad2} to its closed conformation inhibited the heterodimerization between \protein{Cdc20} and \protein{Mad2} and compromised the SAC signaling activity \cite{Ma+Poon2016, Ma+Poon2018, Kim2018}, which can be interpreted by the ``knitting'' model. Second, the binding of \protein{Cdc20} to \protein{Mad1}'s RWD domain is suggested to be important to the SAC signaling activity \cite{Kruse2014, SpMad1, Faesen2017, Ji2017eLife}. One explanation is that the binding may increase the local concentration of \protein{Cdc20}, which is better harnessed by the ``knitting'' model. The anchoring of the basic motif of \protein{Cdc20} to \protein{Mad1}'s RWD domain may even topologically hinder the stringing of \protein{Mad2} that is already in the closed conformation because \protein{Cdc20}'s MIM is between the basic motif and the \textbeta{}-propeller. Third, two recent studies strongly supported the spatio-temporal coupling between \protein{Mad2}'s conformational change and the formation of the \protein{Cdc20}-\protein{Mad2} dimer \cite{BUB1-CDC20-MAD1, Tripartite}.

In this chapter, we intend to prove the "knitting" model by introducing mutations to \protein{Mad1} that theoretically only impair the SAC signaling activity if the "knitting" model is correct. It should be noted that there is a gray area between the two models, given that I-\protein{Mad2} can exist in the solution \cite{I-MAD2}, which may be the fresh \protein{Mad2} conformer coming off the template-based conformational switch, entering the cytosolic pool, and eventually binding to \protein{Cdc20} (becoming C-\protein{Mad2} thereafter). We categorize this situation into the "assembly line" model (also illustrated in \myref{TwoModelCartoons}) because our mutational approach should not impair the SAC signaling activity in this case either (see \myref{LoopDeletionSection}).

\section{The \protein{Mad1}-\protein{Mad2} heterotetramer may adopt a fold-back conformation both \Latin{in vivo} and \Latin{in vitro}}
\label{TwoPossibilities}

For the ``knitting'' model to work, the site of \protein{Mad2}'s conformational switch and \protein{Cdc20}'s MIM have to be positioned close enough. According to the crystal structure, the axial length from the MIM to the start of the loop region is about 5--\SI{6}{nm} while the axial length from the end of the loop region to the anchoring point of \protein{Cdc20} is about \SI{7}{nm} \cite{TemplateModel, Ji2017eLife, BUB1-CDC20-MAD1, Structure1GO4, Structure4DZO}. In contrast, if we model the disordered N-terminus of \protein{Cdc20} as a simple 3-D random walk, we can estimate the root-mean-square distance from \protein{Cdc20}'s BOX1 to its MIM (about 90 residues) to be less than \SI{4}{nm}. Due to steric hindrance and restrictions imposed by the Ramachandran plot, such a simple 3-D random walk model is far from realistic. However, even if we apply a more sophisticated worm-like chain model with a persistence length of 3--\SI{4}{\r{A}} \cite{RandomWalk3D-WormLikeChain}, the root-mean-square distance from \protein{Cdc20}'s BOX1 to its MIM is still only about \SI{5}{nm}. These estimations suggest that additional mechanisms may be at work to position the site of \protein{Mad2}'s conformational switch and \protein{Cdc20}'s MIM closely to facilitate the heterodimerization between \protein{Mad2} and \protein{Cdc20}.

One possibility has been suggested in \cite{BUB1-CDC20-MAD1, Tripartite} that the cooperative binding of \protein{Cdc20} to \protein{Mad1}'s RWD domain and \protein{Bub1} helps to unravel the N-terminus of \protein{Cdc20}. Indeed, if the disordered region from \protein{Cdc20}'s BOX1 to its MIM is fully stretched, it will be more than enough for the MIM of the \protein{Cdc20} anchored to \protein{Mad1}'s RWD domain to engage the site of \protein{Mad2}'s conformational switch. However, there are conflicting claims over whether and where \protein{Bub1} directly binds to \protein{Mad1} \cite{Ji2017eLife, BUB1-CDC20-MAD1, BUB1CD1-MAD1CStructure}. If \protein{Bub1}'s conserved domain 1 (CD1, which is also named conserved motif 1 or CM1) directly interacts with \protein{Mad1}'s consensus RLK motif located in \protein{Mad1}'s C-terminal coiled coil \cite{Ji2017eLife, BUB1CD1-MAD1CStructure} (illustrated in \myref{KnittingModel} as the contact between \protein{Bub1} and the C-terminal coiled coil of \protein{Mad1}), the cooperative binding of \protein{Cdc20} to \protein{Bub1} and \protein{Mad1} may not unravel \protein{Cdc20}'s N-terminus because the segment of \protein{Bub1} from CD1 to ABBA-consensus KEN motifs (which bind to \protein{Cdc20} \cite{BUB1-CDC20-MAD1, CDC20-KEN, ABBA}) is also largely disordered (predicted by AlphaFold2 with low confidence; data not shown).

Another possibility has been suggested in \cite{Structure1GO4, SpMad1} that \protein{Mad1} may position the site of \protein{Mad2}'s conformational switch and \protein{Cdc20}'s MIM closely by assuming a fold-back conformation. As mentioned in the previous section, the secondary structure of \protein{Mad1}'s C-terminal loop region is conserved, whose flexibility may enable such fold-back conformation according to the prediction by the ColabFold advanced algorithm (see \myref{MAD1_ColabFoldPrediction}, which is close to the scenario envisioned by \cite{SpMad1}). If we truncate residues 582--600 of human \protein{Mad1}, which removes the flexible loop and fuses the coiled coils upstream and downstream of the loop region (while preserving the original heptad repeat position designation as assigned by Lupas' method \cite{LupasCOILS} as well as DeepCoil2 \cite{DeepCoil}; data not shown), the resulting loop-deleted \protein{Mad1} (henceforth referred to as \protein{Mad1}\textDelta{}L) homodimer can only assume an extended conformation (see \myref{MAD1DeltaL_ColabFoldPrediction}).

\begin{figure}
    \centering
    \includegraphics[width=0.85\textwidth]{chapters/figures/ColabFoldPrediction.pdf}
    \phantomsubfiglabel{MAD1_ColabFoldPrediction} % subfigure A
    \phantomsubfiglabel{MAD1DeltaL_ColabFoldPrediction} % subfigure B
    \phantomsubfiglabel{MAD1-MAD1DeltaL_ColabFoldPrediction} % subfigure C
    \caption{\textbf{Representative structures of the C-terminus of the \protein{Mad1} homodimer, the \protein{Mad1}\textDelta{}L homodimer, and the \protein{Mad1}-\protein{Mad1}\textDelta{}L heterodimer were predicted by ColabFold.}}
    \noindent\justifying Representative models (out of five models in total) of the C-terminus of the \protein{Mad1} homodimer (A), the \protein{Mad1}\textDelta{}L homodimer (B), and the \protein{Mad1}-\protein{Mad1}\textDelta{}L heterodimer (C) were predicted by ColabFold using the default parameter settings of ColabFold advanced algorithm. The loop region of wildtype \protein{Mad1} is circled out in (A) and (C). The MIM of \protein{Mad1} where C-\protein{Mad2} binds and the RWD domain at the C-terminal end of \protein{Mad1} where the N-terminal BOX1 of \protein{Cdc20} is anchored are also indicated. The C-terminus of the \protein{Mad1} homodimer may assume a fold-back conformation [see the bottom panel of (A)]. In the predicted structure of the \protein{Mad1}-\protein{Mad1}\textDelta{}L heterodimer, the loop region of the wildtype copy introduces a bulge but the overall conformation is extended. 
    \label{ColabFoldPrediction}
\end{figure}

To confirm such fold-back of \protein{Mad1}, we first noted that about 37\% of purified full-length \protein{Mad1}-\protein{Mad2} heterotetramers do not show distinguishable \protein{Mad1}-MIM-\protein{Mad2} and \protein{Mad1}-RWD domain densities as visualized by electron microscopy using metal shadowing \cite{BUB1-CDC20-MAD1}. This may be explained by that this population of \protein{Mad1}-\protein{Mad2} heterotetramers assumes a fold-back conformation. To further support that the \protein{Mad1} may assume a fold-back conformation \Latin{in vivo}, we resorted to the distance-sensitive FRET measurement. We fused the donor fluorophore mNeonGreen to the C-terminal end of \protein{Mad1} (see \myref{TaggingSACProteins}) and inserted the acceptor fluorophore mScarlet-I into the \textbeta{}5-\textalpha{}C loop of \protein{Mad2} \cite{mSI, beta5-alphaCLoop} (the recombinant protein is henceforth referred to as ``\protein{Mad2}$\wedge$mScarlet-I''; see \myref{MAD2InternalTagging_Strategy}). If the heterotetramer always assumes an extended conformation, the distance between the donor and acceptor will be larger than \SI{10}{nm}, which allows no or minimal FRET between the donor \protein{Mad1}-mNG and the acceptor \protein{Mad2}$\wedge$mScarlet-I \cite{Kukreja2020}. However, if the heterotetramer may assume a fold-back conformation, the distance between the donor and acceptor may be reduced to allow for FRET between the donor and the acceptor.

\begin{figure}
    \centering
    \includegraphics[width=0.91\textwidth]{chapters/figures/InternallyTaggedMAD2+NPCFLIM.pdf}
    \phantomsubfiglabel{MAD2InternalTagging_Strategy} % subfigure A
    \phantomsubfiglabel{MAD2InternalTagging_WB} % subfigure B
    \phantomsubfiglabel{MAD2InternalTagging_SACActivity} % subfigure C
    \phantomsubfiglabel{NosiCDC20_FLIM} % subfigure D
    \phantomsubfiglabel{siCDC20_WB} % subfigure E
    \phantomsubfiglabel{siCDC20_FLIM} % subfigure F
    \caption{\textbf{The observation of FRET between \protein{Mad1}-mNG and \protein{Mad2}$\wedge$mScarlet-I at the interphase/prophase NPC suggested that \protein{Mad1}-\protein{Mad2} heterotetramers may assume a fold-back conformation \Latin{in vivo}.}}
    \label{InternallyTaggedMAD2+NPCFLIM}
\end{figure}
\begin{figure}
    \noindent\justifying \textbf{(Caption of \myref{InternallyTaggedMAD2+NPCFLIM} continued from a previous page)} (A) Diagram of the endogenous gene{Mad2} allele and the genome-edited \gene{Mad2}$\wedge$mScarlet-I allele. Boxes 1--5 represent the exons. The regions between these boxes represent the introns. Boxes 2'--5' encode the same peptides as boxes 2--5 respectively, with the introduction of certain silence mutations that make the recombinant \protein{Mad2}$\wedge$mScarlet-I resistant to si\gene{Mad2}. The black ``\textit{P}'' arrow represents the promoter and the 5'-UTR. The black ``\textit{Ter}'' bar represents the 3'-UTR and the polyadenylation signal. The gray ``\textit{Ter}*'' bar represents the polyadenylation signal of rabbit \textbeta{}-globin. The red stop signs represent stop codons. The sequence of the \gene{Mad2}$\wedge$mScarlet-I allele was confirmed by genotyping and Sanger sequencing (data not shown). (B) Immunoblotting showed that \protein{Mad2}$\wedge$mScarlet-I (labeled by an asterisk, with an expected molecular weight of \SI{51.0}{kDa}) was correctly expressed in the heterozygous \gene{Mad2}$\wedge$mScarlet-I HeLa-A12 cell line and was resistant against si\gene{Mad2}. As a comparison, wildtype \protein{Mad2} (labeled by a cruciform with a molecular weight of \SI{23.5}{kDa}) was effectively knocked down by si\gene{Mad2}. The immunoblot against GAPDH served as the loading control. (C) Unsynchronized cells were treated with respective siRNAs for one day, treated with \SI{50}{nM} nocodazole. Each gray dot represents a cell. The total number of cells in each group $N > 140$. Mean values $\pm 95\%$ confidence intervals are overlaid. Unpaired $t$-tests with Welch's correction were performed in Prism 9. Results are representative of two independent experiments. (D) The average lifetime of \protein{Mad1}-mNG in the \gene{Mad1}-mNG or the \gene{Mad1}-mNG/\gene{Mad2}$\wedge$mScarlet-I double genome-edited HeLa-A12 cell  line using a two-component exponential decay to fit the raw data of FLIM. Each dot represents a single cell. Mean values $\pm 95\%$ confidence intervals are overlaid. The total number of cells in each group $N = 7$. Results are representative of two independent experiments. Unpaired $t$-tests with Welch's correction were performed in Prism 9. (E) Unsynchronized HeLa-A12 cells were treated with si\gene{Cdc20} or a control siRNA for \SI{2}{d} and probed for \protein{Cdc20}, \protein{Mad1}, and GAPDH (loading control). (F) Same as (D), except that cells were treated with si\gene{Cdc20} as in (E).
\end{figure}

We confirmed the expression of full-length \protein{Mad2}$\wedge$mScarlet-I in the heterozygous \gene{Mad2}$\wedge$mScarlet-I genome-edited HeLa-A12 cell line (see \myref{MAD2InternalTagging_WB}), wherein the expression level of either \protein{BubR1} or \protein{Cdc20} was not affected. \protein{Mad2}$\wedge$mScarlet-I could support a certain degree of the SAC signaling activity when the endogenous wildtype \protein{Mad2} was knocked down (see \myref{MAD2InternalTagging_SACActivity}). Using fluorescence lifetime imaging microscopy (FLIM) \cite{FLIM}, we quantified a FRET efficiency of about 3\% between \protein{Mad1}-mNG and \protein{Mad2}$\wedge$mScarlet-I at the interphase/prophase NPC in the heterozygous \gene{Mad1}-mNG, \gene{Mad2}$\wedge$mScarlet-I genome-edited HeLa-A12 cell line (see \myref{NosiCDC20_FLIM,FLIMMethod}). We chose to measure FRET at the interphase/prophase NPC to facilitate data collection by the line-scanning confocal microscope and to avoid the potential intermolecular FRET between a donor from one \protein{Mad1}-\protein{Mad2} heterotetramer and an acceptor from another nearby heterotetramer at the corona of a signaling kinetochore. This FRET persisted even when \gene{Cdc20} was knocked down by RNAi (see \myref{siCDC20_WB,siCDC20_FLIM}), which indicates that this FRET does not depend on \protein{Mad1}'s catalytic role but is rather intrinsic to the structural characteristics of the \protein{Mad1}-\protein{Mad2} heterotetramer. Although we still cannot completely rule out potential intermolecular FRET, we confirmed the existence of the anticipated FRET that may result from the fold-back of \protein{Mad1} \Latin{in vivo}.

\section{\protein{Mad1}'s loop region is important to the SAC signaling activity \Latin{in vivo}}
\label{LoopDeletionSection}

We next sought to determine whether the loop region of \protein{Mad1} that likely enables the fold-back conformation is important to the SAC signaling activity \Latin{in vivo}. We integrated the expression cassette of either \protein{Mad1}-mNG or \protein{Mad1}\textDelta{}L-mNG into the genome of HeLa-A12 cells using Cre-\bacterialgene{lox} RMCE (see \myref{Cre-lox}). We then knocked down endogenous \protein{Mad1} in these cells using two siRNAs that target the 3'-UTR of \gene{Mad1} \cite{siMAD1-3UTR} (henceforth collectively referred to as si\gene{Mad1}'s) and induced the expression of \protein{Mad1}(WT/\textDelta{}L)-mNG (si\gene{Mad1}-resistant due to the lack of the endogenous 3'-UTR) by doxycycline. Our genome-edited \gene{Mad1}-mNG HeLa-A12 cell line served as the reference for the endogenous level of \protein{Mad1} in live-cell fluorescence imaging.

We found out that knocking down \gene{Mad1} crippled the SAC signaling activity (see \myref{MAD1Rescue_Main}), although cells with less than 10\% of the physiological level of \protein{Mad1} on average (see \myref{MAD1Rescue_WB}) still arrested in mitosis for hours when treated with \SI{100}{nM} nocodazole (only about two hours less than the control group with a physiological level of \protein{Mad1}). Increasing the dosage of si\gene{Mad1}'s did not further decrease the SAC activity, reinforcing that even a small pool of \protein{Mad1} could sustain a considerable level of SAC signaling activity (see \myref{MAD1Rescue_LimitPushing}).

\begin{figure}
    \centering
    \includegraphics[trim={0 0 0 19cm},clip,width=\textwidth]{chapters/figures/MAD1Rescue.pdf}
    \phantomsubfiglabel{MAD1Rescue_Main} % subfigure A
    \phantomsubfiglabel{MAD1Rescue_LimitPushing} % subfigure B
    \phantomsubfiglabel{MAD1Rescue_HeterodimerizationWB} % subfigure C
    \phantomsubfiglabel{MAD1Rescue_Localization} % subfigure D
    \phantomsubfiglabel{MAD1Rescue_WB} % subfigure E
    \caption{\textbf{The loop region of \protein{Mad1} is critical to the SAC signaling activity in its own right.}}
    \label{MAD1Rescue}
\end{figure}
\begin{figure}
    \noindent\justifying \textbf{(Caption of \myref{MAD1Rescue} continued from a previous page)} (A) The first two columns on the left used the \gene{Mad1}-mNG genome-edited HeLa-A12 cell line which served as a reference of the endogenous level of \gene{Mad1}. \Latin{In situ} tagging of \gene{Mad1} did not affect the 3'-UTR which si\gene{Mad1}'s target. The effectiveness of si\gene{Mad1}'s against the \gene{Mad1}-mNG allele was confirmed by the greatly diminished green channel fluorescence signal (data not shown). The two columns on the right used Cre-\bacterialgene{lox} RMCE HeLa-A12 cell lines which were treated with si\gene{Mad1}'s and induced to express either \protein{Mad1}-mNG or \protein{Mad1}\textDelta{}L-mNG. Each dot represents a cell ($N \geq 50$ in each group). Results were pooled from at least two technical repeats. The mean value $\pm$ the 95\% confidence interval of each group is overlaid. Unpaired $t$-tests with Welch's correction were performed in Prism 9. (B) The conditions of si\gene{Mad1} treatment were (from left to right): untreated, \SI{40}{nM} each for two days (the standard condition used throughout this study), \SI{100}{nM} each for two days, \SI{100}{nM} each in day one and \SI{100}{nM} each again in day two. The \gene{Mad1}-mNG genome-edited HeLa-A12 cell line was used in each group. Each dot represents a cell ($N \geq 145$ in each group). The mean value $\pm$ the 95\% confidence interval of each group is overlaid. Welch's ANOVA test [$W(\text{DF}n, \text{DF}d) = 0.9885 (2.000, 298.9)$, $p = 0.3733$] was performed for the three columns on the right. The ANOVA test and the unpaired $t$-test with Welch's correction were done in Prism 9. (C) Using immunoblotting to evaluate the immunoprecipitation by the mNeonGreen-Trap Agarose. The cruciform symbol represents the endogenous \protein{Mad1} band. The expected molecular weights of the recombinant \protein{Mad1}-mNG, \protein{Mad1}\textDelta{}L-mNG, and mNeonGreen are \SI{110.2}{kDa}, \SI{108.4}{kDa}, and \SI{26.9}{kDa}, respectively. The immunoblot against GAPDH served as the loading control. Blots shown here are from the same immunoprecipitation experiment representative of two independent repeats. (D) The \gene{Mad2}$\wedge$mScarlet-I genome-edited HeLa-A12 treated with si\gene{Mad1}'s and rescued by \protein{Mad1}(WT/\textDelta{}L)-mNG were imaged using wide-field fluorescence microscopy. Cells were arrested at mitosis using a thymidine--nocodazole synchronization protocol (see \myref{WBMethods}). Representative micrographs are shown in the top panel. Maximum $z$-projected green channel images shown here share the same LUT. Maximum $z$-projected red channel images shown here also share the same LUT. Scale bar, \SI{10}{\micro m}. Due to various expression levels of induced \protein{Mad1}(WT/\textDelta{}L)-mNG in different cells, signaling kinetochores were filtered by the localization of \protein{Mad1}(WT/\textDelta{}L)-mNG (with an arbitrary threshold of 1000--\SI{7000}{AU}). Each gray dot represents a single signaling kinetochore ($N \geq 85$ in each group). The mean value $\pm$ the 95\% confidence interval of each group is overlaid. Unpaired $t$-tests with Welch's correction were performed in Prism 9. (E) Knockdown of the endogenous \gene{Mad1} by si\gene{Mad1}'s had an efficiency of over 90\% based on the intensity of the residual \protein{Mad1} band. The cellular abundance of either \protein{BubR1}, \protein{Cdc20}, or \protein{Bub3} was not affected. The immunoblot against GAPDH served as the loading control. Unsynchronized Cre-\bacterialgene{lox} RMCE HeLa-A12 cells were used.
\end{figure}

Surprisingly, \protein{Mad1}\textDelta{}L-mNG had impaired support for the SAC in a dominant-negative manner: cells treated with si\gene{Mad1}'s and rescued by a physiological level of \protein{Mad1}\textDelta{}L-mNG arrested in mitosis for a significantly shorter duration than cells which were not rescued. As a comparison, wildtype \protein{Mad1}-mNG fully restored the SAC signaling activity (see \myref{MAD1Rescue_Main}). One possible explanation was that \protein{Mad1}\textDelta{}L-mNG might dimerize with the remaining endogenous \protein{Mad1} and restrict its structural flexibility. Structural prediction of the heterodimer between wildtype \protein{Mad1} and \protein{Mad1}\textDelta{}L showed that the loop region of the wildtype copy introduced a bulge but could not enable a fold-back conformation of the heterodimer due to stiffness of the fused \textalpha{}-helices of the \protein{Mad1}\textDelta{}L counterpart (see \myref{MAD1-MAD1DeltaL_ColabFoldPrediction}). To prove this hypothesis, we pulled down doxycycline-induced \protein{Mad1}(wildtype/\textDelta{}L)-mNG from lysates of HeLa-A12 cells in which endogenous \protein{Mad1} was not knocked down. We found that endogenous \protein{Mad1} was also pulled down by both \protein{Mad1}-mNG and \protein{Mad1}\textDelta{}L-mNG, but not by mNeonGreen alone (see \myref{MAD1Rescue_HeterodimerizationWB}). We further confirmed that \protein{Mad1}\textDelta{}L-mNG did not cause defects in the localization of the \protein{Mad1}\textDelta{}L-\protein{Mad2} heterotetramer (see \myref{MAD1Rescue_Localization}) or the expression of \protein{BubR1}, \protein{Cdc20}, or \protein{Bub3} (see \myref{MAD1Rescue_WB}). \Latin{In vitro} reconstitution data from our collaborators (not shown here) also suggested that truncating the loop region of \protein{Mad1} reduced the formation rate of \protein{Cdc20}-\protein{Mad2}. Therefore, although the results of our knockdown-rescue experiments were hindered by the incomplete knockdown of the endogenous \protein{Mad1}, all evidence combined suggested that the loop region of \protein{Mad1} is critical to the SAC signaling activity in its own right.

\section{\protein{Mad1}(Lmut) can fully support the SAC signaling activity}
% The function of \protein{Mad1}'s loop region does not depend on the potential phosphorylation of any serine or threonine residues within it

Many possibilities could underlie the observation that the deleted segment encompassing residues 582--600 in \protein{Mad1}\textDelta{}L is important to the SAC signaling activity. It is known that S598 can be phosphorylated by \protein{Mps1} \Latin{in vitro} \cite{MAD1_pS598}. There could also be a certain unknown protein-protein interaction associated with the loop region of \protein{Mad1} that is important to the SAC. To prove that it is the structural flexibility provided by the loop region that is critical to the SAC signaling activity, we tested two different artificial flexible loops to replace the original segment encompassing residues 582--600: ``AL11'' and ``Lmut'' (see \myref{AL11+Lmut_Lupas+DeepCoil2}). Both of them consist of 19 amino acid residues like the original one. AL11 is a previously characterized flexible linker composed of 11 alanine residues, 7 glycine residues, and 1 threonine residue \cite{AL11}. In Lmut, serine and threonine residues of the original segment are mutated into alanine and valine residues, respectively. The amino acids between the two prolines are adjusted into mostly alanine-glycine-alanine repeats while the proline residues themselves and their N-terminal neighboring residues are preserved. % AL11 starts with glycine-threonine at the N-terminus like the endogenous segment.

\begin{figure}
    \centering
    \includegraphics[width=0.95\textwidth]{chapters/figures/AL11+Lmut.pdf}
    \phantomsubfiglabel{AL11+Lmut_Lupas+DeepCoil2} % subfigure A
    \phantomsubfiglabel{AL11+Lmut_Rescue} % subfigure B
    \caption{\textbf{\protein{Mad1}(AL11)-mNG cannot fully rescue the SAC signaling activity but \protein{Mad1}(Lmut)-mNG can.}}
    \noindent\justifying (A) The figure shows coiled-coil predictions of human wildtype \protein{Mad1} (top), \protein{Mad1}(AL11) (middle), \protein{Mad1}(Lmut) (bottom) by the two algorithms as in \myref{MAD1_MIM-RLK_Lupas+DeepCoil2}. The top panel is reproduced from \myref{MAD1_MIM-RLK_Lupas+DeepCoil2}. Segments encompassing residues 582--600 are highlighted by gray boxes. (B) The first two columns on the left used the \gene{Mad1}-mNG genome-edited HeLa-A12 cell line which served as a reference of the endogenous level of \gene{Mad1}. The AllStars negative control siRNA was added to the first group and si\gene{Mad1}'s were added to the second group. The two columns on the right used the Cre-\bacterialgene{lox} RMCE HeLa-A12 cell lines which were treated with si\gene{Mad1}'s and induced to express either \protein{Mad1}(AL11)-mNG or \protein{Mad1}(Lmut)-mNG. Each dot represents a cell ($N \geq 75$ in each group). Results were pooled from at least two technical repeats. The mean value $\pm$ the 95\% confidence interval of each group is overlaid. Unpaired $t$-tests with Welch's correction were performed in Prism 9.
    \label{AL11+Lmut}
\end{figure}

Both \protein{Mad1}(AL11) and \protein{Mad1}(Lmut) were predicted to have a similar coiled-coil propensity profile to that of the endogenous \protein{Mad1} (see \myref{AL11+Lmut_Lupas+DeepCoil2}). Surprisingly, cells treated with si\gene{Mad1}'s and rescued by protein{Mad1}(AL11)-mNG had a lower SAC signaling activity than wildtype cells did (see \myref{AL11+Lmut_Rescue}). As a comparison, \protein{Mad1}(Lmut)-mNG fully restored the SAC signaling activity. We hypothesized that because proline residues may assume the \Latin{cis}-conformation at a higher propensity than many other amino acid residues at equilibrium \cite{Ramachandran1976}, they might facilitate the proper fold-back of \protein{Mad1}.

The fact that Lmut can functionally replace the original segment encompassing residues 582--600 revealed that the SAC likely relies on the structural flexibility but not phosphorylation or unknown protein-protein interactions (other than those that rely solely on the prolines and their N-terminal neighboring residues; see \myref{Chapter4Discussions} for more details) associated with the loop region.

\section{The structural flexibility of \protein{Mad1} enabled by its loop region facilitates the ``knitting'' of the MCC}
\label{FinalKnittingModel}

Based on previous literature and our data, we proposed the following mechanistic model of the catalytic role of \protein{Mad1} in the assembly of \protein{Cdc20}-\protein{Mad2} (see \myref{KnittingModel}). \protein{Mad1} in its fold-back conformation physically positions the MIM of \protein{Cdc20} and \protein{Mad2} closely, facilitating the spatio-temporal coupling of \protein{Mad2}'s conformational switch and the assembly of the \protein{Cdc20}-\protein{Mad2} heterodimer. The avidity binding between the scaffold protein \protein{Mad1} (in its fold-back conformation) and \protein{Cdc20}-\protein{Mad2} energetically favors the formation of the \protein{Mad1}RWD-\protein{Cdc20}-\protein{Mad2}-\protein{Mad1}MIM complex. \protein{Mad1} may switch back to an extended conformation (probably in an ATP-dependent manner), breaking such avidity binding and promoting the release of \protein{Cdc20}-\protein{Mad2} into the cytosol.

\begin{figure}
    \centering
    \includegraphics[width=0.75\textwidth]{chapters/figures/KnittingModel.pdf}
    \caption{\textbf{``Knitting'' model: \protein{Mad1}'s structural flexibility may facilitate the spatio-temporal coupling of \protein{Mad2}'s conformational switch and the assembly of the \protein{Cdc20}-\protein{Mad2} heterodimer.}}
    \noindent\justifying The two solid black arrows indicate the formation and release of the \protein{Cdc20}-\protein{Mad2} heterodimer, respectively. T461-phosphorylated CD1 of \protein{Bub1} (458--476) interacts with \protein{Mad1}'s consensus RLK motif located within the coiled coil leading up to the RWD domain \cite{Ji2017eLife, BUB1CD1-MAD1CStructure}. Additionally, the C-terminus of \protein{Bub1}'s CD1 contacts the RWD domain of the opposite \protein{Mad1} copy to whose consensus RLK motif the CD1 interacts with \cite{BUB1CD1-MAD1CStructure}. \protein{Bub1} interacts with \protein{Cdc20} through multiple motifs cooperatively. For example, \protein{Bub1}'s ABBA motif (527--532) binds between Blades 2 and 3 of \protein{Cdc20}'s seven-bladed WD40 fold and \protein{Bub1}'s consensus KEN box (C-terminal to the ABBA motif) likely binds to the center of the top surface of \protein{Cdc20}'s WD40 \cite{BUB1-CDC20-MAD1, CDC20-KEN, ABBA}. \protein{Cdc20} also binds to \protein{Mad1} and \protein{Bub1} cooperatively \cite{BUB1-CDC20-MAD1, Tripartite}. However, whether \protein{Bub1} and \protein{Mad1} cooperate to unravel \protein{Cdc20} and scaffold the assembly of \protein{Cdc20}-\protein{Mad2} is unclear (refer to the two possibilities in \myref{TwoPossibilities}). The proposed avidity binding between the scaffold protein \protein{Mad1} (in its fold-back conformation) and \protein{Cdc20}-\protein{Mad2} involves the \protein{Mad1}RWD-\protein{Cdc20} interaction, the ``safety belt''-like binding between \protein{Cdc20} and \protein{Mad2}, as well as the homodimerization of the \protein{Mad2} in the \protein{Cdc20}-\protein{Mad2} heterodimer and the \protein{Mad2} in the \protein{Mad1}-\protein{Mad2} heterotetramer \cite{TemplateModel, Structure1GO4, beta5-alphaCLoop, I-MAD2, Ji2017eLife, BUB1-CDC20-MAD1}.
    \label{KnittingModel}
\end{figure}

However, there are many missing links towards the ``knitting'' model. For example, do \protein{Bub1} and \protein{Mad1} cooperate to unravel \protein{Cdc20} and scaffold the assembly of \protein{Cdc20}-\protein{Mad2}? Does \protein{Mad1} switch between an extended conformation and the fold-back conformation at a physiologically meaningful rate \Latin{in vivo} and does this switching cycle correlate with the knitting of a \protein{Cdc20}-\protein{Mad2} heterodimer? Does \protein{Mad1}'s conformational switch require energy input? Do the proline residues simply serve to break the coiled coil and shape the flexible loop of \protein{Mad1}, or do they play a more complex role in the proper fold-back of \protein{Mad1}? Future studies are needed to fill these gaps between current experimental evidence and the ``knitting'' model.

\section{Discussions}
\label{Chapter4Discussions}

In this study, we showed for the first time that the \protein{Mad1}-\protein{Mad2} heterotetramer can assume a fold-back conformation both \Latin{in vitro} and \Latin{in vivo}. Our preliminary data indicate that the structural flexibility is enabled by a flexible loop in the C-terminus of \protein{Mad1}, whose secondary structure is conserved. This loop region is important to the SAC signaling activity both \Latin{in vitro} and \Latin{in vivo} and its role does not depend on any potential phospho-regulation of the serine/threonine residues within. We proposed a ``knitting'' model to explain how the structural flexibility of \protein{Mad1}-\protein{Mad2} heterotetramer may facilitate the spatio-temporal coupling between \protein{Mad2}'s conformational change and the formation of \protein{Cdc20}-\protein{Mad2}. It should be noted that even in the absence of \protein{Cdc20}, O-\protein{Mad2} can bind to the \protein{Mad1}-\protein{Mad2} heterotetramer, complete the conformational switch, and dissociate as C-\protein{Mad2} \Latin{in vitro} \cite{Yang2008}. Finding out whether \protein{Cdc20} increases the rate of \protein{Mad2}'s conformational switch will help us further understand the mechanism of the catalysis.

\protein{Mad1} has a long half-life under normal conditions \cite{MAD1MAD2Half-life}. And like \protein{Bub1} \cite{Raaijmakers2018, RZZ-MAD1vsBUB1-MAD1_2018, siROD_Zhang2019}, even a small pool of \protein{Mad1} (at less than 10\% on average of its physiological concentration in our knockdown experiments) can maintain a considerable level of SAC signaling activity in nocodazole-treated cells. Future studies should consider combining RNAi (or induced knockout of \gene{Mad1}) with induced acute degradation of \protein{Mad1} proteins to reduce the contribution of the remaining endogenous \protein{Mad1} homodimer to the SAC signaling activity and to minimize the chance of heterodimerization between the remaining endogenous \protein{Mad1} and the rescuing \protein{Mad1} variants in such knockdown-rescue experiments. %(AID, Trim-Away, etc. for a nuclear protein during the interphase/prophase).

Given the critical role of \protein{Mad1}'s structural flexibility enabled by its loop region, it would be interesting to replace the flexible loop with a turn to lock \protein{Mad1} in the fold-back conformation and see how the SAC signaling activity is affected. Another way to advance our understanding is to investigate whether the two pools of \protein{Mad1} (adopting either the fold-back or extended conformation) inter-convert at a physiologically meaningful rate in the cell using single-molecular approaches. Even though the two proline residues (P585 and P596) in \protein{Mad1}’s loop region may be important to the SAC signaling activity \Latin{in vivo}, no \protein{Mad1}-interacting protein with peptidylprolyl cis-trans isomerase activity has been identified in the PrePPI database using the gene ontology enrichment analysis tool from the PANTHER Classification System as of March 2022 \cite{PrePPI, PANTHER}. Additionally, it might be worth finding out whether the equilibrium between the two conformations in the cell is the same as purified \protein{Mad1}-\protein{Mad2} heterotetramer \Latin{in vitro}, which would tell us if the conformation distribution is under active regulation in the cell that costs energy \cite{MAD2Dynamics}. % However, this does not completely rule out the possibility because the interaction between \protein{Mad1} and a peptidylprolyl isomerase might be transient.

Two missense variants (D587N and A593V) related to \protein{Mad1}’s loop region were recorded in the Genomic Data Commons Data Portal as of March 2022 \cite{GDC}, but the impact of both point mutations is predicted to be benign. Therefore, the physiological impact of potential mutations in \protein{Mad1}’s loop region at the organism level is unclear. It would be interesting to see the physiological impact of introducing point mutations (for example, the multiple proline residues) in \protein{Mad1}’s loop region in various model systems.

In addition to \protein{Cdc20}, closed \protein{Mad2} also interacts with many other proteins (including \protein{Mad1}, \protein{Sgo2} \cite{SGO2-MAD2}, the insulin receptor \cite{MCC_IREndocytosis}, and \protein{Kif20a} \cite{KIF20A-MAD2}), likely by a similar ``safety belt'' mechanism \cite{Structure1GO4}. One question that comes up naturally is whether the same catalytic mechanism (spatio-temporal coupling) similarly applies to how \protein{Mad2} binds to other proteins (or even more generally, whether the same catalytic mechanism applies to how other HORMA domain proteins bind to other proteins \cite{HORMAReview})? One interesting finding is that the S214A mutation in human \protein{Mad1} impairs the homodimerization of \protein{Mad1} as well as the interaction between \protein{Mad1} and \protein{Mad2} \cite{ATMPhosphorylatesMad1S214}. S214A is unlikely to affect the binding of \protein{Mad2} to \protein{Mad1}'s MIM directly, given the structure of the \protein{Mad1}-\protein{Mad2} heterotetramer \cite{Structure1GO4} and the fact that S214 and MIM are over 300 amino acids apart. This suggested that the homodimerization of \protein{Mad1} might facilitate the binding of \protein{Mad2} to \protein{Mad1}. One possibility is that one copy of \protein{Mad1} may trans-activate the binding of \protein{Mad2} to the other copy of \protein{Mad1} in the \protein{Mad1}-\protein{Mad2} heterotetramer. Future experiments are needed to elucidate the structural and catalytic basis of how \protein{Mad2} ``buckles up'' its binding partners, which may unveil how \protein{Mad2} regulates mitosis beyond the assembly of the MCC \cite{Separase-SGO2-MAD2, KIF20A-MAD2}. % is it over-reading?

% Two possibilities of how CDC20 MIM is brought to close proximity with MAD2 are not mutually exclusive.
\Latin{In vitro} reconstitution data from our collaborators suggested that the critical role of the flexibility of \protein{Mad1} in scaffolding the spatio-temporal coupling relies on \protein{Bub1}. In the absence of \protein{Bub1} in the reactions, the formation rates of \protein{Cdc20}-\protein{Mad2} were the same for both \protein{Mad1} and \protein{Mad1}\textDelta{}L (data not shown). However, it is known that the assembly of the MCC also occurs during the interphase and prophase \cite{PremitoticMCC}. There has been no report on \protein{Bub1}'s localization at the NPC where the \protein{Mad1}-\protein{Mad2} heterotetramer is predominantly localized during the interphase and prophase. Therefore, either the flexibility of \protein{Mad1} alone scaffolds the coupling at the NPC or there may be a nucleoporin that functions similarly to \protein{Bub1}. Interestingly, the nuclear basket protein \protein{Tpr}, which directly associates with the \protein{Mad1}-\protein{Mad2} heterotetramer during the interphase and prophase \cite{TPR-MAD1_Lee2008}, is predicted to bind to \protein{Cdc20} directly in the PrePPI database \cite{PrePPI}. Future studies should look into how the \protein{Mad1}-\protein{Mad2} heterotetramer may catalyze the formation of the \protein{Cdc20}-\protein{Mad2} dimer at the NPC during the interphase and prophase.

\section{Materials and Methods}

For methods of cell culture and Cre-\bacterialgene{lox} RMCE, see \myref{CellCulture+RMCE_Methods}. Time-lapse live-cell imaging was performed on an ImageXpress Nano Automated Imaging System as described in \myref{chpt3ImagingMethods}. Wide-field, $z$-stack fluorescence imaging used in the quantification of localization of \protein{Mad1}(WT/\textDelta{}L)-mNG and \protein{Mad2}$\wedge$mScarlet-I at signaling kinetochores was also as described in \myref{chpt3ImagingMethods}.

\subsection{Generating the \gene{Mad2}$\wedge$mScarlet-I genome-edited HeLa-A12 cell line}

The gRNA used in the integration of the coding sequence of \protein{Mad2}$\wedge$mScarlet-I (intron-free, stop codon-containing, and si\gene{Mad2}-resistant by the introduction of silent mutations) and the polyadenylation signal of rabbit \textbeta{}-globin after the first exon of the endogenous \gene{Mad2} gene was \Oligo{ucgcgcaggccaauauaucg}. Synthesis of the sgRNA and assembly of the \Latin{Sp}Cas9-sgRNA RNP complex were as described in \myref{CRISPRMethods}. Plain or \gene{Mad1}-mNG genome-edited HeLa-A12 cell lines were co-transfected with the RNP complex and linearized pCC35, sorted, and validated as described in \myref{CRISPRMethods}. A successfully edited \gene{Mad2}$\wedge$mScarlet-I allele encodes an internally-tagged \protein{Mad2} protein, wherein wildtype \protein{Mad2} and mScarlet-I are separated by short flexible linkers (\Peptide{AGSGSGGAS} between the S114 of \protein{Mad2} and the N-terminus of mScarlet-I; \Peptide{GTGAGSA} between the C-terminus of mScarlet-I and the A115 of \protein{Mad2}).

\subsection{RNAi}

The two siRNAs targeting the 3'-UTR of \gene{Mad1} were from \cite{siMAD1-3UTR}. They were applied to unsynchronized cells at a concentration of \SI{40}{nM} each for two days before imaging or collecting cells for immunoblotting unless specified otherwise. The sense-strand sequence of si\gene{Cdc20} was \Oligo{GGAGCUCAUCUCAGGCCAU} \cite{siCDC20}, which was applied at a concentration of \SI{40}{nM} for two days before FLIM or immunoblotting. The sense-strand sequence of si\gene{Mad2} was \Oligo{GGAAGAGUCGGGACCACAGUU} \cite{BubR1MitosisTurnover}, which was applied at a concentration of \SI{40}{nM} for one day before imaging or immunoblotting. Desalted siRNAs modified by double-deoxythymidine overhangs at 3'-ends of both strands were synthesized by Sigma. The AllStars Negative Control siRNA was used as the control. All siRNAs were transfected into the cells via Lipofectamine RNAiMAX following manufacturer’s instructions.

\subsection{Fluorescence lifetime imaging microscopy (FLIM)}
\label{FLIMMethod}

Similar to \myref{FCSMethods}, all FLIM data were collected on an Alba v5 Laser Scanning Microscope, connected to an Olympus IX81 inverted microscope main body [equipped with a UPLSAPO60XW objective (1.2 NA)]. A Fianium WL-SC-400-8 laser with an acousto-optic tunable filter was used to generate excitation pulses at a wavelength of \SI{488}{nm} and a frequency of about \SI{20}{MHz}. Excitation light was further filtered by a Z405/488/561/635rpc quadband dichroic mirror. The emission light of the green channel was redirected by a 562 longpass dichroic mirror (FF562-Di03, Semrock), filtered by an FF01-531/40-25 filter, and finally detected by an SPCM-AQRH-15 avalanche photodiode. The time-correlated single photon counting module to register detected photon events to excitation pulses was SPC-830. Data acquisition was facilitated by VistaVision.

The instrument response function of the green channel was measured using Rose Bengal (which has poor excitation efficiency at \SI{488}{nm}) freshly dissolved to \SI{1}{\micro M} in \SI{5.6}{M} KI solution \cite{RoseBengal}. The emission light was redirected by a 562 longpass dichroic mirror and filtered by an FF01-582/75-25 filter (Semrock). The data analysis pipeline (implemented in MATLAB) developed in this study is publicly available.

To demonstrate how fluorescence lifetime measurements can quantify the FRET efficiency, consider donor fluorophores with a lifetime of $\tau_0$. In the absence of acceptor fluorophores, the exponential decay $D_0$ of donor fluorescence after the pulse excitation at $t = 0$ is

\begin{equation*}
    D_0(t) = C e^{-\frac{t}{\tau_0}}
\end{equation*}

\noindent and the total fluorescence is

\begin{equation*}
    S_0 = \int_0^{+\infty}D_0(t) dt = C\tau_0,
\end{equation*}

\noindent wherein $C$ is a constant determined by the excitation and detection condition, the total number and properties of fluorophores, and the imaging setup. Without altering any of these conditions, in the presence of acceptor fluorophores and FRET, the longer a donor fluorophore stays in the excited state, the higher the chance FRET may have occurred (note: this is not a rigorous statement because fluorescence emission and FRET quenching are independent stochastic processes and an excited fluorophore can only relax through one route). Suppose that the timing of FRET follows an exponential distribution with a probability density function of

\begin{equation*}
    f(t) \coloneqq \dfrac{1}{\tau'} e^{-\frac{t}{\tau'}}.
\end{equation*}

\noindent The probability that FRET does not happen before $t_0$ will be

\begin{equation*}
    P(t > t_0) = \int_{t_0}^{+\infty} f(t) dt = e^{-\frac{t_0}{\tau'}}.
\end{equation*}

\noindent Excited fluorophores can either take the FRET quenching route or the fluorescence emission route to relax to the ground state (note: a fluorophore may also relax through other ways but the fact that these routes are independent stochastic processes means that it does not affect the following conclusion). Therefore, in the presence of acceptor fluorophores and FRET, the decay $D$ of donor fluorescence becomes

\begin{equation*}
    D(t_0) =  D_0(t_0) \cdot P(t > t_0) = C e^{-\frac{t_0}{\tau_0}} \cdot e^{-\frac{t_0}{\tau'}} \coloneqq C e^{-\frac{t_0}{\tau}},
\end{equation*}

\noindent wherein

\begin{equation*}
    \tau = \dfrac{\tau_0\tau'}{\tau_0 + \tau'}.
\end{equation*}

Thus, the total fluorescence becomes $S =  C\tau$. Therefore,

\begin{equation*}
    \text{FRET efficiency} \coloneqq \frac{S_0-S}{S_0} = \frac{\tau_0-\tau}{\tau_0}.
\end{equation*}

Because the fluorescence lifetime in the absence of quenching is an intrinsic property of a mature fluorescent protein (under a certain temperature) \cite{FLTIsIntrinsic}, the equation above greatly simplifies experiments to measure the FRET efficiency. This equation still applies even if the fluorescence decay has to be fitted by a multi-component exponential decay, as long as the fluorescence lifetime is an average weighted by the corresponding $C$ of each component.

\subsection{Immunoprecipitation using mNeonGreen-Trap}
\label{mNGTrapMethod}

HeLa-A12 cells integrated with the expression cassette (under the regulation of a TRE) of either mNeonGreen, \protein{Mad1}-mNG, or \protein{Mad1}\textDelta{}L-mNG were induced to express the ectopic recombinant protein by \SI{0.1}{\micro g/ mL} doxycycline (for two days until being harvested) and arrested at mitosis using a thymidine--nocodazole synchronization protocol described in \myref{WBMethods}. Cells were harvested by mitotic shake-off, washed once by PBS, pelleted down by centrifugation at 200--\SI{500}{g} for \SI{3}{min}, snap-frozen in liquid nitrogen, and stored at \SI{-80}{\celsius} before the immunoprecipitation experiment.

On the day of the immunoprecipitation experiment, cells were thawed on ice and lysed in the IP lysis buffer [\SI{75}{mM} HEPES-HCl (pH 7.5 at \SI{4}{\celsius}), \SI{150}{mM} KCl, 10\% (by volume) glycerol, \SI{1.5}{mM} \ch{MgCl2}, \SI{1.5}{mM} EGTA, and 1\% (by mass) CHAPS, a zwitterionic detergent] supplemented before usage with \SI{1}{mM} PMSF, the cOmplete\texttrademark{} EDTA-free Protease Inhibitor Cocktail, Phosphatase Inhibitor Cocktail IV (RPI), and a phosphatase inhibitor cocktail (\SI{1}{mM} \ch{Na4P2O7}, \SI{0.1}{mM} \ch{Na3VO4}, \SI{5}{mM} NaF, and \SI{2}{mM} sodium \textbeta-glycerophosphate). For \SI{1}{mg} of wet cell pellet, \SI{40}{\micro L} of \SI{4}{\celsius} IP lysis buffer was added, yielding a total protein concentration of about \SI{5.6}{mg/mL} (if cells were lysed completely). Resuspended cells were rotated for \SI{30}{min} at \SI{4}{\celsius} and then centrifuged at \SI{18000}{g} for \SI{20}{min} at \SI{4}{\celsius}. \SI{600}{\micro L} of supernatant was subsequently cleared by \SI{50}{\micro L} of equilibrated control agarose beads (ChromoTek) to reduce non-specific bindings, rotating for \SI{45}{min} at \SI{4}{\celsius}. The mixture was centrifuged at \SI{2000}{g} for \SI{5}{min} at \SI{4}{\celsius}. \SI{580}{\micro L} of pre-cleared supernatant was then mixed with \SI{30}{\micro L} of equilibrated mNeonGreen-Trap Agarose (nta-20, ChromoTek) and rotated for \SI{1}{h} at \SI{4}{\celsius}. These beads were then pelleted down at \SI{2000}{g} for \SI{5}{min} at \SI{4}{\celsius} and the supernatant was removed. The beads were further washed for four times (rotated for \SI{5}{min} at \SI{4}{\celsius} and then pelleted down at \SI{2000}{g} for \SI{5}{min} at \SI{4}{\celsius}) using \SI{1}{mL} of the IP wash buffer [\SI{75}{mM} HEPES-HCl (pH 7.5 at \SI{4}{\celsius}), \SI{150}{mM} KCl, 10\% (by volume) glycerol, \SI{1.5}{mM} \ch{MgCl2}, and \SI{1.5}{mM} EGTA] each time. The beads were transferred to a fresh tube before the last wash to avoid the non-specific binding of proteins to the wall of the tube. Finally, $2\times$ Laemmli buffer supplemented with \textbeta-mercaptoethanol was added to the beads. Samples were boiled in a boiling water bath for \SI{10}{min} before being subjected to SDS-PAGE and immunoblotting analysis.

\subsection{Immunoblotting}

Samples for immunoblotting were prepared as described in \myref{WBMethods,mNGTrapMethod}. Primary antibodies (and their working dilution factors by volume) used included anti-\protein{BubR1} (Bethyl Laboratories A300-995A-M, $1 : 1000$), anti-\protein{Cdc20} (Santa Cruz Biotechnology sc-13162, $1 : 200$), anti-\protein{Mad2} (Bethyl Laboratories A300-301A-M, $1 : 330$), anti-GAPDH (Proteintech 60004-1-Ig, $1 : 5000$), anti-\protein{Mad1} (GeneTex GTX109519, $1 : 2000$ in \myref{siCDC20_WB} and PLA0092, $1 : 1000$ in \myref{MAD1Rescue_HeterodimerizationWB,MAD1Rescue_WB}), anti-mNeonGreen (Cell Signaling Technology 53061S, $1 : 100$), and anti-\protein{Bub3} (Sigma-Aldrich B7811, $1 : 500$). Digital images were acquired using an Azure C600 or 600 imaging system (Azure Biosystems).