\protein{Mad1} has a long half-life under normal conditions \cite{MAD1MAD2Half-life}. And like \protein{Bub1} \cite{Raaijmakers2018, RZZ-MAD1vsBUB1-MAD1_2018, TraceBub1}, even a small pool of \protein{Mad1} (at less than 10\% of its physiological concentration in our knockdown experiments) can maintain a considerable SAC activity in nocodazole-treated cells. Future studies should consider combining RNA interference (or induced knockout of \gene{Mad1}) with acute degradation of \protein{Mad1} proteins for a more profound effect in such knockdown-rescue experiments. %(AID, Trim-Away, etc. for such a nuclear protein during the interphase/prophase).

In addition to \protein{Cdc20}, closed \protein{Mad2} also interacts with many other proteins (including \protein{Mad1}, \protein{Sgo2} \cite{SGO2-MAD2}, the insulin receptor \cite{MCC_IREndocytosis}, and \protein{Kif20a} \cite{KIF20A-MAD2}), likely by a similar ``safety belt'' mechanism \cite{Structure1GO4}. One question that comes up naturally is whether the same catalytic mechanism (the spatio-temporal coupling between \protein{Mad2}'s conformational change and the formation of the \protein{Cdc20}-\protein{Mad2} dimer) similarly applies to how \protein{Mad2} binds to other interactors (or even more generally, does the same catalytic mechanism apply to other HORMA domain proteins \cite{HORMAReview})? One interesting finding is that the S214A mutation in human \protein{Mad1} impairs the homodimerization of \protein{Mad1} as well as the interaction between \protein{Mad1} and \protein{Mad2} \cite{ATMPhosphorylatesMad1S214}. S214A is unlikely to affect the binding of \protein{Mad2} to \protein{Mad1}'s MIM directly, given the structure of the \protein{Mad1}-\protein{Mad2} heterotetramer \cite{Structure1GO4} and the fact that S214 and MIM are over 300 amino acids apart. This suggested that the homodimerization of \protein{Mad1} might facilitate the binding of \protein{Mad2} to \protein{Mad1}. Future experiments are needed to elucidate the structural and catalytic basis of how \protein{Mad2} ``buckles up'' its binding partners.

% Do the two MAD1 conformation pools interconvert constantly at a meaningful rate (and if so, where does the energy come from? does BUB1/a peptidylprolyl isomerase facilitate it?) [No MAD1 interaction has been found in the PrePPI database within the class of GO:0005528 (FK506-binding) or the class of GO:0003755 (peptidyl-prolyl cis-trans isomerase activity) as of June 1, 2021. However, this does not completely rule out the possibility because the interaction between MAD1 and a peptidylprolyl isomerase might be transient.] Consider using protein design (HLH/HTH/leucine zipper/zinc finger) to create a new MAD1 which is exclusively in the fold-back conformation without affecting its interactions with other proteins…

Two missense variants (D587N and A593V) related to \protein{Mad1}’s loop region were recorded in the Genomic Data Commons Data Portal as of March 2022 \cite{GDC}, but the impact of both point mutations is predicted to be benign. Therefore, the physiological impact at the organism level of potential mutations in \protein{Mad1}’s loop region, which could have shed some light on its functions and evolution, is unclear. It would be interesting to see the physiological impact of introducing point mutations (for example, the multiple proline residues) in \protein{Mad1}’s loop region in various model systems.